app/models/project/submission.rb

```diff
@@ -0,0 +1,33 @@
+# == Schema Information
+# Schema version: 20200509082917
+#
+# Table name: project_submissions
+#
+#  id            :integer          not null, primary key
+#  project_id    :integer
+#  user_id       :integer
+#  resource_type :string
+#  resource_id   :integer
+#  skipped       :boolean
```

Comment:
I'm thinking `skipped` should be ephemeral and stored in a session-based queue, rather than creating empty submissions. Maybe remove it for now and if we decide we should use `Project::Submission` to handle skipping we can add it in later?